main.js

```diff
@@ -0,0 +1,120 @@
+const posts = []
+const postList = document.getElementById('post-list');
+const postBtn = document.getElementById('post-btn');
+const lineBreak = document.createElement('br');
+
+function makePost() {
+  const postInput = document.getElementById('post-input');
+  const posterName = document.getElementById('name-input');
+  const postId = Math.random().toString(36).substr(2, 9); 
+  if (postInput.value === "" || posterName.value === '')  {
```

Comment:
you are not trimming the input, this allows to make a post with empty spaces on any input.